spec/performance/pre_deploy_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "net/http"
+require "rspec-benchmark"
+
+TEST_SAMPLE_COUNT=5
+
+RSpec.configure do |config|
+  config.include RSpec::Benchmark::Matchers
+end
+
+def get(path)
+  uri = URI(ENV.fetch("CUSTOM_HOST_NAME", "http://localhost:3001"))
+  uri.path = path
+  Net::HTTP.get(uri)
+end
+
+tests = {
+  "/api/v3/recruitment_cycles/2020/providers/U80" => 300,
+  "/api/v3/recruitment_cycles/2020/providers/U80/courses/2P3K" => 300,
```

Comment:
maybe 
these 2 should be configurable so we can have a separated azure tasks additional capabilities

Reply:
Not sure what you mean. Configurable in what way? We're pretty much aiming for a proof of concept here so as we can get it set up and then iterate on them along the way.

Reply:
```
CUSTOM_HOST_NAME=https://api2.publish-teacher-training-courses.service.gov.uk /
PROVIDER_CODE=U80 /
COURSE_CODE=2P3K /
bundle exec rspec spec/performance/pre_deploy_spec.rb
```

Something like so...

Reply:
Right... I see what you mean. I think the plan is to run these on the integration test data we add once that's in there. I'll add the other env variables so as it's more flexible. Good idea.